projects/zeek/build.sh

```diff
@@ -74,3 +74,8 @@ for f in ${fuzzers}; do
 
     fuzzer_count=$((fuzzer_count + 1))
 done
+
+# Normally, base-builder/compile copies sources for use in coverage reports,
```

Comment:
can you please put this comment and command under `

`if [ "${SANITIZER}" = 'coverage' ]; then`